qst/src/main/java/io/deephaven/qst/table/TicketTable.java

```diff
@@ -33,7 +33,9 @@ public static TicketTable of(byte[] ticket) {
      *
      * @param ticket the ticket string
      * @return the ticket table
+     * @deprecated prefer to be explicit and either use {@link #of(byte[])} or {@link #fromQueryScopeField(String)}
      */
+    @Deprecated
```

Comment:
@rbasralian brought up point; should we prefer to add a more explicitly named `fromUtf8` so users can still have an un-deprecated call to this feature? I do think that `of(String)` made it too easy to mess up, but there may still be general value in the function?

Reply:
I'd vote no, String.toBytes(StandardCharsets.UTF8) or whatnot is right there (assuming they want utf8 and not ascii etc...).

At least not without some feedback requesting this.